Amplify/Categories/DataStore/Model/Internal/Schema/ModelField+Association.swift

```diff
@@ -88,28 +88,48 @@ import Foundation
 ///   directly by host applications. The behavior of this may change without warning.
 public enum ModelAssociation {
     case hasMany(associatedFieldName: String?, associatedFieldNames: [String] = [])
-    case hasOne(associatedFieldName: String?, targetNames: [String])
-    case belongsTo(associatedFieldName: String?, targetNames: [String])
+    case hasOne(associatedFieldName: String?, associatedFieldNames: [String] = [], targetNames: [String])
+    case belongsTo(associatedFieldName: String?, associatedFieldNames: [String] = [], targetNames: [String])
```

Comment:
It appears we have defined `belongsTo` to hold the associatedFieldName but the codegen output never generates this information (see https://github.com/aws-amplify/amplify-codegen/pull/541/files#diff-eaba881c8c73aea316d5f63356d638f20a3bf275c7a4b66d8e97e1070c2de177 line 634)